src/main/webapp/jsp/instructorHome.jsp

```diff
@@ -6,7 +6,7 @@
 <c:set var="jsIncludes">
   <script type="text/javascript" src="/js/instructorHome.js"></script>
 </c:set>
-<ti:instructorPage pageTitle="TEAMMATES - Instructor" bodyTitle="Instructor Home" jsIncludes="${jsIncludes}">
+<ti:instructorPage pageTitle="Home" bodyTitle="Instructor Home" jsIncludes="${jsIncludes}">
```

Comment:
Just highlighting that if we drop the `pageTitle` attribute, the `bodyTitle` should just be `Home` as the title would already read `Home [Instructor]`.